src/server/migrations/5_create_user_belong_project_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE users__project (
+  users_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  project_id INTEGER REFERENCES project(id) ON DELETE CASCADE
+);
+
+ALTER TABLE users__project
+ADD CONSTRAINT PK_users__project PRIMARY KEY (users_id, project_id);
```

Comment:
Oh, and it's also handier to just write the primary key after the fields:
```
CREATE TABLE foo (
a bunch of fields,
PRIMARY KEY (users_id, project_id)
);
```

but since that doesn't affect the table, let's skip that change.